test/Features/contentTranslation/NodeReferences.feature

```diff
@@ -7,10 +7,10 @@ Feature: Node References
   Background:
     Given I am logged in as a user with the "administer entity xliff" permission
     And "page" content:
-      | title                              | field_long_text                    | promote |
-      | English node host page title       | English node host body text.       | 1       |
-      | English node child page title      | English node child body text.      | 1       |
-      | English node grandchild page title | English node grandchild body text. | 1       |
+      | title                              | field_long_text                    | promote | status |
+      | English node host page title       | English node host body text.       | 1       | 1       |
+      | English node child page title      | English node child body text.      | 1       | 1       |
+      | English node grandchild page title | English node grandchild body text. | 1       | 1       |
```

Comment:
It has to do with the expectation that workbench will have completed whatever it needs to do. In this case we are not looking at the published status AFTER doing some other operation. I only changed the ones that really had to change since the "And "page" content:" version is so much more readable.